content/blog/entries/2022-12-07-berkeley-quantifying/contents.lr

```diff
@@ -0,0 +1,441 @@
+- Significant duplication issue (resolved).
+
+
+#### Google Custom Search API
+
+- Programmable Search Engine only reaches a subset of Google's website. The
+  impact is not significant (then, further resolved via sampling frame
+  adjustments in PSE).
+- Accidentally used deprecated operators and parameters, causing faithfulness
+  problems (resolved).
+
+
+#### YouTube Data API
+
+- API has maximum response value on total count of YouTube videos, causing
+  severe underestimate.
+  - Resolved via implementing custom granularity on data to enable honest
+    response, conserve development cost, and introduce imputations in
+    visualization.
+
+
+### Expanding the Dataset
+
+Here are reasons and efforts of dataset expansion on platforms that received
+more data:
+
+
+#### Google Custom Search API
+
+- Revised Data Sampling process to solve EDA-discovered inaccuracies.
+- For expanding the horizons of CC product usage analyses upon past boundaries,
+  where visualization was only conducted to compare cross-product performance,
+  I incorporated further CC-product usage data across temporal axis and
+  geographical demographics.
+
+
+#### YouTube Data API
+
+- Revised Data Sampling process to solve EDA-discovered inaccuracies.
+- To perform unprecedented analyses on media-specific time-respective
+  developments of CC options on popular platforms, YouTube's CC-licensed
+  video count across two-month periods.
+- Introduced imputation to alleviate unresolvable capped responses from YouTube
+  and mitigate developmental cost in response to Youtube API's capping
+  behaviour.
+
+
+## Visualization
+
+
+### Philosophies and Principles
+
+The visualizations of Quantifying the Commons is to be communicative and
+exhibitory.
+
+Some new aesthetics and principles we adopted (as a response to enhancement of
+prior efforts) are to:
+- Present length in place of area for comprehensibility
+- Analyze product development beyond license-wise comparisons
+- Utilize colors for presenting data inclinations via works in Pandas, Seaborn,
+  NumPy, Geopandas, and SpaCy
+
+
+### Exhibiting a Selection of Visualizations
+
+
+#### Diagram 1C
+
+Trend Chart of Creative Commons Usage on Google
+![Trend Chart of Creative Commons Usage on Google](diagram_1c.png)
+
+There are now **more than 2.7 Billion webpages protected by Creative Commons**
+indexed by Google!
+
+
+#### Diagram 2
+
+Heatmap on density of CC-licensed Google indexed webpages over country
+![Heatmap on density of CC-licensed Google indexed webpages over country](diagram_2.png)
+
+Particularly, **Western Europe and Americas enjoy a much robust use** of
+Creative Commons document in terms of quantity. A Development in Asia and
+Africa should be encouraged.
+
+
+#### Diagram 3C
+
+Barplot for number of webpages protected by six primary CC licenses
+![Barplot for number of webpages protected by six primary CC licenses](diagram_3c.png)
+
+We that **Attribution** (BY) and **Attribution-Nonderivative (BY-ND) are
```

Comment:
missing word, 

`We ____ that`
 
perhaps,
`found`, `determined`, or `concluded`?